.semaphore/semaphore.yml

```diff
@@ -0,0 +1,30 @@
+version: v1.0.1
+name: Apple/Mojave Based Pipeline
+agent:
+  machine:
+    type: a1-standard-4
+    os_image: macos-mojave-xcode11
+blocks:
+  - name: macOS Builds
+    task:
+      jobs:
+        - name: calculator_simple
+          commands:
+            - brew install make
+            - brew install gcc
+            - brew install gtk+3
+            - brew install gtkmm3
+            - checkout
+            - ls -alF
+            - cd ./calculator_simple/build/macos/
+            - gmake
+        - name: test_install
+          commands:
+            - brew install make
+            - brew install gcc
+            - brew install gtk+3
+            - brew install gtkmm3
+            - checkout
+            - ls -alF
+            - cd ./calculator_simple/build/macos/
+            - gmake
```

Comment:
This should be 2 blocks to reuse the installation by storing then restoring (loading) the cached configuration:
https://docs.semaphoreci.com/guided-tour/caching-dependencies/

That was what was missing in the previous commits which had these as separate blocks originally -- no use of cache, before.

Reply:
Split into 2 blocks in d51574a

Reply:
Ok, nope, totally misread (read: did not actually read) the documentation and apparently the cache restore is more for like `pip` or `yarn` or `npm` or whatever, not like actually the environment, which is probably more the purpose of like a docker container, but there seems to be no official macOS docker image, so we're back to two options:

1. Undo the changes in d51574a and just leave this as it was, albeit seemingly redundant
1. Try to mess around with the caching to add the brew directories, since there seems to be TravisCI documentation on achieving a similar goal.

I was also thinking maybe we could try using YAML references, like with the Merge operator (`<<`), but then I double-checked and merging is only supported for maps (key-value pairs) not sequences (lists), so there's no built-in way in the current YAML standard to reference a "base" sequence and add new elements to it ... which would make choice 1 nicer in terms of not leaving around bug-bait with the redundant setup code ... but that seems to be the fastest, sanest approach here.

So, I say we go with 1 and then circle back in a later pull-request if we wanna go for 2 and build out more features from SemaphoreCI.

Reply:
Ok, now actually resolved with 0f789b2 per the above discussion/discovery.